portfolio/src/main/java/com/google/sps/servlets/StringServlet.java

```diff
@@ -32,6 +32,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         //return the arraylist as json
```

Comment:
Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace